src/stock-user-role-scopes/add-stock-user-scope/add-stock-user-role-scope.component.tsx

```diff
@@ -253,28 +230,6 @@ const AddStockUserRoleScope: React.FC<AddStockUserRoleScopeProps> = ({
     });
   };
 
-  const onEnableDescendantsChanged = (event: ChangeEvent<HTMLInputElement>) => {
```

Comment:
i think this is necessary. find use and code 
https://github.com/openmrs/openmrs-module-stockmanagement/blob/b3ca338f551b0561c3beb42fc659252534b9e6f8/owa/src/features/userRoleScope/EditUserScope.tsx#L180

Reply:
It wasn't used in that file, otherwise it would have caused an error

Reply:
@jabahum this might have been needed in the module but in this new code it wasn't being used and we need to clean up such to avoid having a lot of code which we are not using, we can always put it back if we need it.